### ISSUE TITLE
When playing a game in play mode, the dialog that should have been updated are not reflected.

### ISSUE THREAD

**Issue by amenonegames** [YarnSpinnerTool/YarnSpinner-Unity#212]
The yarnfile was updated on my budy's mac pc(apple sillicon). And same PC played the yarn project.
But the dialog that should have been updated are not reflected.

When I updated it on my Windows PC it worked fine.
When I re-created yarnproject from yarnfile on budy's mac PC , it worked fine.
It also worked fine on MacPC when I re-created the project anew.

This error occur when same yarn project used , but yarn file is updated , I thout.

It appears that the added lines are ignored. And various error occured cause delete line.

The same problem seems to have occurred only in ver 2.2.2 or later, and reverting to 2.2.1 worked fine in the same environment.

According to my colleague, there was a message that the line ID could not be found.

**Comment by McJones** [comment]
Can you give us some more detail as to the yarn files that caused this issue?
We are having a hard time reproducing this on our end.
Thanks.

**Comment by mchambers** [comment]
I'm also experiencing this, with a very basic Yarn file. It seems confined to Play mode in the Editor. If I play, then make changes to the Yarn, save, then play again, lines get skipped, new nodes don't appear at all and character names will disappear.

Like the original filer I am also going between two machines but I'm not sure if that makes a difference.

Unity 2020.3.48f1 and Yarn Spinner 2.2.5+134 installed via Package Manager.

**Comment by McJones** [comment]
Have been periodically trying to reproduce this on and off for a while now.
I am still unable to do so, and there hasn't been any response from the original poster with more information.
At this stage I am going to close this issue as I can't reproduce it.

My guess is that files aren't being save correctly either by not actually saving the file after editing or because of domain reload being disabled.
If this issue reappears please open a new issue.

**Comment by hsandt** [comment]
Reproduced with Unity 6000.0.12f1 + YarnSpinner 2.4.2

Setup:

- Packages/rpg-framework/Sample Assets/Data/Quest/YarnQuests/Sample YarnProject Quests.yarnproject
- Packages/rpg-framework/Sample Assets/Data/Quest/YarnQuests/YarnScripts/Sample YarnScript Quest_1.yarn

The Yarn Project is looking for Source Files: YarnScripts/**/*.yarn

Dialogue script before change:

```
title: Quest_1
tags:
---
Narrator: Oh, hello!
    -> Hi, where am I?
        Narrator: You're in Unity!
            -> Oh.
                <<jump Oh>>
            -> How did I get here?
                <<jump Unity>>
===

title: Oh
---
Narrator: Yeah, fun, right?
===

title: Unity
---
Narrator: Someone read the Beginner's Guide!
===
```

Dialogue script after change:
```
title: Quest_1
tags:
---
Narrator: Battle start!
start_battle 0
===
```

I was initially not reloading domain so I tried to enable Reload Scene, Reload Domain, and both, to no avail. I still see "Oh, hello!" every time.

Maybe the fact that the script is inside Packages folder matters? I'll try again with scripts in actual game folder.
